okhttp-tests/src/test/java/okhttp3/internal/http/CookiesTest.java

```diff
@@ -163,6 +163,28 @@ public void testNetscapeResponse() throws Exception {
     assertEquals("a=android; b=banana", request.getHeader("Cookie"));
   }
 
+  @Test public void receiveAndSendMultipleCookies() throws Exception {
+    MockWebServer server = new MockWebServer();
+    server.enqueue(new MockResponse()
+        .addHeader("Set-Cookie", "a=android")
+        .addHeader("Set-Cookie", "b=banana"));
+    server.enqueue(new MockResponse());
+    server.start();
+
+    CookieManager cookieManager = new CookieManager(null, ACCEPT_ORIGINAL_SERVER);
```

Comment:
This `CookieManager`'s implementation is difference between openJDK and the Android JDK.
openJDK's `CookieManager#get(URI, Map<String, List<String>>)`[1] return a map whose cookie's entry is a List contains each cookie. But the Android JDKs `CookieManager#get(URI, Map<String, List<String>>)`[2] return a map whose cookie's entry is a List with only one string. This string was composed of several cookies.
So If you write a test for openJDK, this test can pass because
```
    // copied from javaNetCookieJar.java#L70
    List<Cookie> cookies = null;
    for (Map.Entry<String, List<String>> entry : cookieHeaders.entrySet()) {
      String key = entry.getKey();
      if (("Cookie".equalsIgnoreCase(key) || "Cookie2".equalsIgnoreCase(key))
          && !entry.getValue().isEmpty()) {
        for (String header : entry.getValue()) {
          if (cookies == null) cookies = new ArrayList<>();
          cookies.addAll(decodeHeaderAsJavaNetCookies(url, header));
        }
      }
    }
```
use `for each` to invoke `decodeHeaderAsJavaNetCookies(HttpUrl, String)` for each cookie. But Android's won't, because Android's only invoke `for each` once, and use `decodeHeaderAsJavaNetCookies(HttpUrl, String)` to parse string which was composed of several cookies.

I guess this test only works in Android if Android N use openJDK's implementation.

[1] http://grepcode.com/file/repository.grepcode.com/java/root/jdk/openjdk/8u40-b25/java/net/CookieManager.java/#195
[2] https://android.googlesource.com/platform/libcore/+/f7ab2bc37debba91864bfec6572a3e7bbe994c58/luni/src/main/java/java/net/CookieManager.java#120